### PR TITLE
feat(translation): Add Galician translation

### PR DESCRIPTION
**Pull request by siderio2**
Add ga.js file to .../languages, plus modify some related files

BREAKING CHANGE: New translation

#421

### CODE REVIEW

**Comment by siderio2** [comment]
Two things:
1. I have finally used the white flag emoji while the flag system is not changed, but I'm not sure if I'm using it right: all flags included until now have a shortcode like ':flag-pt:', but I have seen that for the white flag it is  ':flag_white:'. Does it work??
2. And I'm asking this because I'm not able to run the app locally on my computer, so it would be nice if someone can check it. I'm going to gitter to ask what am I doing wrong with the local deployment.

**Comment by lex111** [comment]
> I have finally used the white flag emoji while the flag system is not changed, but I'm not sure if I'm using it right: all flags included until now have a shortcode like ':flag-pt:', but I have seen that for the white flag it is ':flag_white:'. Does it work??

In fact, it is not yet clear what to do with the flags. But in any case, it does not affect your translation in any way, so do not worry about it yet.

> And I'm asking this because I'm not able to run the app locally on my computer, so it would be nice if someone can check it. I'm going to gitter to ask what am I doing wrong with the local deployment.

Of course, let's try to figure out what's wrong.

File: src/auth/screens/language-settings.js [hunk 1]
```diff
@@ -34,4 +34,9 @@ export default [
     emojiCode: ':flag-ru:',
     name: 'Русский',
   },
+  {
+    code: 'gl',
+    emojiCode: ':flag_white:',
```
Comment (housseindjirdeh) [inline src/auth/screens/language-settings.js]: The list of emoji codes we can use are [here](). You're right, `flag_white` doesn't exist but you can use `:waving_white_flag:` to have this :white_flag:

**Comment by housseindjirdeh** [comment]
@lex111 feel free to do a pass through, other than changing the emoji code to `:waving_white_flag:` it looks good to me.

@siderio2 we would love to help you get the app working on your machine as well. Please let us know whatever issue you might be seeing and we'll help :raised_hands:

**Comment by siderio2** [comment]
Well, it took me more than expected, but I finally managed to run the app on my machine. I already changed the flag shortcode, but it doesn't work on my phone (Redmi note 4 with android 6.0). And also I see a string not translated in options: 'Open in Browser', but I can't find it...

**Comment by housseindjirdeh** [comment]
Woo awesome glad to hear you got it working mate :tada: 

Yeah so the `Open in Browser` string was merged just recently so you'll have to rebase/merge from master to see the line in `en.js`. And hmm interesting how the icon doesn't show. Don't worry about that, @lex111 and I are already discussing replacing all those emojicodes with circular images anyway :)

File: ios/GitPoint.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -71,6 +70,7 @@
 		DC5C35ED1E37AD1800F3F526 /* SimpleLineIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = DC5C35E31E37AD1800F3F526 /* SimpleLineIcons.ttf */; };
 		DC5C35EE1E37AD1800F3F526 /* Zocial.ttf in Resources */ = {isa = PBXBuildFile; fileRef = DC5C35E41E37AD1800F3F526 /* Zocial.ttf */; };
 		EB0F176BE52B4561B9FF07AB /* libReactNativeConfig.a in Frameworks */ = {isa = PBXBuildFile; fileRef = D7043BDB577E427391ECFBB0 /* libReactNativeConfig.a */; };
+		CBC20ECDD7FF46A49DF646D6 /* Feather.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 20A722EC74A74B50BD5DCCC9 /* Feather.ttf */; };
```
Comment (housseindjirdeh) [inline ios/GitPoint.xcodeproj/project.pbxproj]: Have noticed this locally when linking (I'm assuming RNE has a new icon font called `Feather`?) Not sure why I just didn't push it to master but perfectly okay to have it in this PR

**Pull request merged**